### PR TITLE
Add Wrapper 1155-to-20

### PR DESCRIPTION
**Pull request by davidalbela**
Enable custom wrapped ERC-20 tokens with `tokenName` and `tokenSymbol`. The decimals will be the same number as the collateralToken of the given positionId (tokenId).

- Update 1155-to-20 contract address.
- Add `tokenBytes` parameter on ConditionalTokens and Wrapper115 services.
- Add 1155-to-20-helper dependeny.
- Add `wrappedTokenAddress` attribute on Positions.
- Replace the `wrappedToken` entity assotiacion with Positions into value of the last wrapped token on `wrappedTokenAddress` attribute. 

Requires a subgraph update and re-sync. See https://github.com/gnosis/hg-subgraph/pull/44

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -36,10 +36,10 @@ REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_RINKEBY='0x3D00D77ee771405628a4bA4913175E
 REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_GANACHE='0xcfeb869f69431e42cdb54a4f4f105c19c080a601'
 REACT_APP_REALITY_CONTRACT_ADDRESS_FOR_XDAI='0x79e32aE03fb27B07C89c0c568F80287C01ca2E57'
 
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_MAINNET='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_RINKEBY='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_GANACHE='0xDE6943f3717738038159a406FF157d4eb3238c1B'
-REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_XDAI='0xDE6943f3717738038159a406FF157d4eb3238c1B'
+REACT_APP_WRAPPED_1155_FACTORY_CONTRACT_ADDRESS_FOR_MAINNET='0x6846a6e06B88c1449Db0B3cc555d4638eDC1bF37'
```
Comment (denisgranha) [inline .env.example:39]: It's missing  https://etherscan.io/address/0x6846a6e06B88c1449Db0B3cc555d4638eDC1bF37

Comment (davidalbela) [reply .env.example:39]: wrapped 1155 it's not deployed yet. Once we deploy the contract on mainnet will generate the same address based on the bycode and the singleton contract.

Comment (davidalbela) [reply .env.example:39]: The env.example can be updated one the contract PR is merged and deployed.

Comment (davidalbela) [reply .env.example]: The new wrapper address is `0xc14f5d2B9d6945EF1BA93f8dB20294b90FA5b5b1`

**Pull request merged**